spec/definitions/EnterpriseV1.yaml

```diff
@@ -0,0 +1,90 @@
+required:
+  - id
+  - name
+  - short_code
+type: object
+properties:
+  id:
+    title: Id
+    description: Id of the enterprise
+    type: string
+    format: uuid
+    readOnly: true
+  name:
+    title: Name
+    description: Name of the enterprise
+    type: string
+    maxLength: 255
+    minLength: 1
+  short_code:
+    title: Short code
+    description:  A short 10-char representation of the enterprise, separated by hyphens. Field is optional, as system will auto-generate a value if none is provided.
+    type: string
+    maxLength: 10
+    minLength: 1
+  registered_name:
+    title: Registered name of the enterprise
+    description: Registered Name of the enterprise
+    title: Registered name
+    type: string
+    maxLength: 255
+    minLength: 1
+  registered_address:
+    title: Registered address
+    description: Registered address of the enterprise.
+    type: string
+    max_length: 255
+    minLength: 1
+  location:
+    title: Enterprise location
+    description: City, State, Country location of the enterprise
```

Comment:
Specify whether it is city or state, which will helpful for marketing team. Better store city, state and country in separate columns.

Reply:
I checked with @jerynmathew he said we can keep it as it is for now.